corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_advanced.html

```diff
@@ -85,16 +85,14 @@ <h4 class="modal-title">{% trans "Remove Form Action?" %}</h4>
                         <span class="help-block" data-bind="text: validate_subcase"></span>
                     </div>
                 </div>
-                {% if show_custom_ref %}
-                    <div class="form-group" data-bind="if: subcase">
-                        <label class="control-label col-sm-2">
-                            {% trans "Parent reference ID" %}
-                        </label>
-                        <div class="col-sm-4">
-                            <input type="text" class="form-control" data-bind="value: case_index.reference_id"/>
-                        </div>
+                <div class="form-group" data-bind="if: subcase">
```

Comment:
any clue as to why this wasn't showing up for either Clayton or me even though we both have this flag turned on?